cmd/bpf2go/gen/output_test.go

```diff
@@ -81,3 +82,23 @@ func TestPackageImport(t *testing.T) {
 var typesEqualComparer = cmp.Comparer(func(a, b btf.Type) bool {
 	return a == b
 })
+
+func TestCustomIdentifier(t *testing.T) {
+	var buf bytes.Buffer
+	called := false
+	identifier := func(input string) string {
+		called = true
+		return ebpfInt.Identifier(input)
+	}
+	args := GenerateArgs{
+		Package:    "foo",
+		Stem:       "bar",
+		ObjectFile: "frob.o",
+		Output:     &buf,
+		Programs:   []string{"do_thing"},
+		Identifier: identifier,
+	}
+	err := Generate(args)
+	qt.Assert(t, qt.IsNil(err))
+	qt.Assert(t, qt.IsTrue(called))
```

Comment:
`Called` feels a little artificial, can we use `strings.ToUpper` as `Identifier` and check that `buf` contains `DO_THING` instead?